lib/Commands/NinjaBuildCommand.cpp

```diff
@@ -1774,14 +1798,22 @@ int commands::executeNinjaBuildCommand(std::vector<std::string> args) {
     }
   }
 
+  SmallString<128> current_dir;
```

Comment:
same as above, `PATH_MAX`?

Reply:
Lev & I already discussed this, paths can exceed path max so I think 256 is fine above. This one here isn't perf sensitive so I think we are ok as is.... if we wanted to clean it up later we should move these constants out to a shared file, but not germane to this PR.